src/components/team-logo/index.tsx

```diff
@@ -0,0 +1,23 @@
+import type { QwikIntrinsicElements } from "@builder.io/qwik";
+import { component$ } from "@builder.io/qwik";
+
+const TEAM_CITY_MAP = {
+  "Atlanta Hustle": "ATL",
+};
+
+export type TeamLogoProps = Omit<
+  QwikIntrinsicElements["img"],
+  "src" | "alt"
+> & { team: keyof typeof TEAM_CITY_MAP };
+
+export const TeamLogo = component$(({ team, ...props }: TeamLogoProps) => {
+  const city = TEAM_CITY_MAP[team];
+
+  return (
+    <img
```

Comment:
Eventually we should download all these logos and serve them ourselves as static assets. But for now we just yoink them from the AUDL website.